src/legacy/core_plugins/data/public/query/query_bar/components/query_string_input.test.tsx

```diff
@@ -90,7 +90,7 @@ function wrapQueryStringInputInContext(testProps: any, storage?: any) {
   );
 }
 
-describe('QueryStringInput', () => {
+describe.skip('QueryStringInput', () => {
```

Comment:
@lizozom I'm not sure if you're the right person to ping about this (blame `git blame` 😅), but I had to skip these tests as they were producing unhandled promise rejections due to an uncaught exception.